examples/v1beta1/taskruns/alpha/object-param-result.yaml

```diff
@@ -23,13 +23,25 @@ spec:
           IMAGE_URL: {type: string}
           IMAGE_DIGEST: {type: string}
     steps:
-      - name: echo-object-params
+      - name: validate-object-params
         image: bash
         args: [
-          "echo",
-          "--url=$(params.gitrepo.url)",
-          "--commit=$(params.gitrepo.commit)"
+          "$(params.gitrepo.url)",
+          "$(params.gitrepo.commit)"
         ]
+        script: |
+          if [[ $1 != "github.example.com" ]]; then
+              echo "Want: github.example.com, Got: $1"
+            exit 1
+          else
+            echo "validate the url successfully"
```

Comment:
nit: validate the `params.gitrepo.url` successfully as well as the `sha`~ Thank you!

Reply:
sure, thanks!